src/app_engine/constants.py

```diff
@@ -11,9 +11,9 @@
 
 LOOPBACK_CLIENT_ID = 'LOOPBACK_CLIENT_ID'
 
-TURN_BASE_URL = 'https://computeengineondemand.appspot.com'
-TURN_URL_TEMPLATE = '%s/turn?username=%s&key=%s'
-CEOD_KEY = '4080218913'
+TURN_BASE_URL = 'https://networktraversal.googleapis.com'
```

Comment:
Actually it follows the http://tools.ietf.org/html/draft-uberti-rtcweb-turn-rest-00#section-2 draft. I know the new turn service do not but maybe it's easier to keep it close to what others might use?